### PR TITLE
[MISC] Fail with an error message when X-Forwarded-* headers are miss…

### PR DESCRIPTION
**Pull request by clems4ever**
…ing.

### CODE REVIEW

File: internal/handlers/handler_register_u2f_step1_test.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		FindIdentityVerificationToken(gomock.Eq(token)).
+		Return(true, nil)
+
+	s.mock.StorageProviderMock.EXPECT().
+		RemoveIdentityVerificationToken(gomock.Eq(token)).
+		Return(nil)
+
+	SecondFactorU2FIdentityFinish(s.mock.Ctx)
+
+	// Return 200 KO
```
Comment (james-d-elliott) [inline internal/handlers/handler_register_u2f_step1_test.go]: Insignificant Typo.

File: internal/handlers/handler_register_u2f_step1_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (james-d-elliott) [inline internal/handlers/handler_register_u2f_step1_test.go]: Insignificant Typo. Should be "// Return 200 OK".

File: internal/handlers/handler_register_u2f_step1_test.go (hunk 1 above)
Comment (james-d-elliott) [reply internal/handlers/handler_register_u2f_step1_test.go]: Resolving this in favor of the more specific one. There are a few instances of this.

File: internal/handlers/handler_register_u2f_step1_test.go (hunk 2 above)
Comment (james-d-elliott) [reply internal/handlers/handler_register_u2f_step1_test.go]: There are actually quite a few, probably just a copy paste.

Comment (clems4ever) [reply internal/handlers/handler_register_u2f_step1_test.go]: I've just removed them.

**Pull request merged**